### ISSUE TITLE
app-crypt/qdigidoc4-4.2.1: * Applying sandbox-compat.patch ... 1 out of 1 hunk FAILED -- saving rejects to file common/CMakeLists.txt.rej

### ISSUE THREAD

**Issue by lkraav** [open-eid/gentoo#34]
`sandbox-compat.patch` needs some review action.

```
>>> Unpacking source...
>>> Unpacking qdigidoc4-4.2.1.tar.gz to /mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/work
>>> Source unpacked in /mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/work
>>> Preparing source in /mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/work/qdigidoc4-4.2.1 ...
 * Applying sandbox-compat.patch ...
1 out of 1 hunk FAILED -- saving rejects to file common/CMakeLists.txt.rej                                                                                                                                                             [ !! ]
 * ERROR: app-crypt/qdigidoc4-4.2.1::esteid failed (prepare phase):
 *   patch -p1  failed with /mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/files/sandbox-compat.patch
 *
 * Call stack:
 *               ebuild.sh, line  124:  Called src_prepare
 *             environment, line 3236:  Called cmake-utils_src_prepare
 *             environment, line 2030:  Called default_src_prepare
 *      phase-functions.sh, line  937:  Called __eapi6_src_prepare
 *             environment, line 1362:  Called eapply '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/files/sandbox-compat.patch'                                                                                                       
 *             environment, line 2215:  Called _eapply_patch '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/files/sandbox-compat.patch'                                                                                                
 *             environment, line 2153:  Called __helpers_die 'patch -p1  failed with /mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/files/sandbox-compat.patch'                                                                         
 *   isolated-functions.sh, line  119:  Called die
 * The specific snippet of code:
 *              die "$@"
 *
 * If you need support, post the output of `emerge --info '=app-crypt/qdigidoc4-4.2.1::esteid'`,
 * the complete build log and the output of `emerge -pqv '=app-crypt/qdigidoc4-4.2.1::esteid'`.
 * The complete build log is located at '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/temp/build.log'.
 * The ebuild environment file is located at '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/temp/environment'.
 * Working directory: '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/work/qdigidoc4-4.2.1'
 * S: '/mnt/datapool/tmp/portage/app-crypt/qdigidoc4-4.2.1/work/qdigidoc4-4.2.1'

>>> Failed to emerge app-crypt/qdigidoc4-4.2.1, Log file:
```

**Comment by monoidic** [comment]
This should fix it: #35

**Comment by urmet** [comment]
whoops, had some untested commits in my local repo that got pushed when fixing #33 
sorry :)
